src/Project/Common/code/App_Config/Include/Project/Common.Website.Serialization.config

```diff
@@ -1,57 +1,60 @@
+
+            <include name="Project.Templates" database="master" path="/sitecore/templates/Project/Habitat Sites">
+              <exclude children="true">
+                <except name="Common"/>
+              </exclude>
+            </include>
+
+            <include name="System.Workflows" database="master" path="/sitecore/system/Workflows/Demo Workflow" />
+            <include name="Feature.Articles.Templates" database="master" path="/sitecore/templates/Feature/Habitat Sites/Articles" />
+            <include name="Feature.Navigation.Templates" database="master" path="/sitecore/templates/Feature/Habitat Sites/Navigation" />
+
+            <include name="Settings" database="master" path="/sitecore/system/Settings/Feature/Habitat Sites" />
+            <include name="Feature.PageContent.Templates" database="master" path="/sitecore/templates/Feature/Habitat Sites/Page Content" />
+            <include name="Feature.PageContent.Renderings" database="master" path="/sitecore/layout/Renderings/Feature/Habitat Sites/Page Content" />
+
+
+            <include name="Content" database="master" path="/sitecore/content/Habitat Sites">
+              <exclude children="true" />
+            </include>
+            <include name="Themes" database="master" path="/sitecore/media library/Themes/Habitat Sites">
+              <exclude database="master" childrenOfPath="/sitecore/media library/Themes/Habitat Sites/Habitat Home Store"  />
+            </include>
+            <include name="Media" database="master" path="/sitecore/media library/Project/Habitat Sites/shared" />
+            <include name="Languages" database="master" path="/sitecore/system/Languages" />
+
+            <!-- SXA Extensions -->
+            <include name="RenderingVariants.WrappedSlide" database="master" path="/sitecore/system/Settings/Foundation/Experience Accelerator/Rendering Variants/Rendering Variants/Page Content/Wrapped Slide" />
+
+            <!-- SXA Overrides -->
+            <include name="SXA.CoreSripts.Typeahead" database="master" path="/sitecore/media library/Base Themes/Core Libraries/scripts/typeahead" />
+
+            <!-- Validation rules -->
+            <include name="ValidationRules.RichText" database="master" path="/sitecore/system/Settings/Validation Rules/Field Types/Rich Text" />
+            <include name="ValidationRules.Global" database="master" path="/sitecore/system/Settings/Validation Rules/Global Rules" />
+
+            <!-- Experience Editor buttons -->
+            <include name="WebEdit.Hero.EditHeroBackground" database="core" path="/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Edit Hero Background" />
+            <include name="WebEdit.Author.AssociateAuthor" database="core" path="/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Associate Author" />
+
+            <!-- Stackla -->
+            <include name="Stackla.Renderings" database="master" path="/sitecore/layout/Renderings/Stackla" />
```

Comment:
Should we have the require:stackla in the integrations to avoid having the need for Stackla as a module?
Is this possible given the way the rest of the content is created?
If this doesn't impact the unicorn sync when the module is not installed then you can disregard.

Reply:
I added the Stackla rendering to the project, so it's always there and does not cause a broken rendering on content items if the module isn't installed. 
The rendering will not render until the module is installed and configured. And it will not fail during Unicorn sync since it's a regular Sitecore rendering. There are no content items in source control that use Stackla templates or are otherwise dependent on the module being installed, so there's no need at the moment for a require:stackla switch.